### PR TITLE
docs: improve contributing guide

### PR DESCRIPTION
**Pull request by Parul1606**
## Related Issue
Closes Issue #1917 

## Description

As per the requirement, I have added two readme points."

## Screenshots

image

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -94,15 +104,15 @@ pnpm dev
 - `git remote add upstream https://github.com/priyankarpal/ProjectsHut.git`
```
Comment (priyankarpal) [inline contributing.md:104]: remove the line

File: contributing.md [hunk 2]
```diff
@@ -59,31 +59,41 @@ git clone https://github.com/<YOUR-USERNAME>/ProjectsHut.git
 cd ProjectsHut
 ```
 
-3.  Install dependencies
+3. Add a reference(remote) to the original repository.
+```
+git remote add upstream https://github.com/<YOUR_USERNAME>/ProjectsHut.git
```
Comment (priyankarpal) [inline contributing.md:64]: change the line to 
```suggestion
git remote add upstream https://github.com/priyankarpal/ProjectsHut.git
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: contributing.md (hunk 2 above)
Comment (Parul1606) [reply contributing.md:64]: Done

File: contributing.md (hunk 1 above)
Comment (Parul1606) [reply contributing.md:104]: Okay..Removed👍🏼

**Review by priyankarpal — APPROVED** [review]

**Pull request merged**